library/src/main/java/com/novoda/downloadmanager/Batch.java

```diff
@@ -9,7 +9,7 @@
     private final String title;
     private final List<String> fileUrls;
 
-    private Batch(DownloadBatchId downloadBatchId, String title, List<String> fileUrls) {
+    Batch(DownloadBatchId downloadBatchId, String title, List<String> fileUrls) {
```

Comment:
We use a `Builder` to create this normally. We should make this package as it is only accessed internally anyway. The `Builder` is just a utility to make this easier.